src/components/BrosweArticles.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+
+export default function BrowseArticles() {
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [articles, setArticlesInfo] = useState([]);
+  useEffect(() => {
+    fetch(`https://jrm-nc-news.onrender.com/api/articles`)
+      .then((response) => response.json())
+      .then((response) => {
+        setArticlesInfo(response);
```

Comment:
I'd recommend separating this out into a separate file and creating a base URL for axios. Let me know if you have any questions about this 👍